thoth/common/workflows.py

```diff
@@ -460,6 +460,10 @@ def submit_workflow(
 
         return wf.name
 
+    def get_total_workflows(self, workflow_namespace: str) -> int:
+        """Get the total number of workflows in a given namespace."""
+        return len(self.api.list_namespaced_workflows(workflow_namespace).items)
```

Comment:
Do we want to have just "pending" workflows?

Reply:
I don't think the API supports this

Reply:
> I don't think the API supports this

Can we do it programmatically on the library side then?